e2e/shortcut-action/shortcut-action.e2e-spec.ts

```diff
@@ -0,0 +1,119 @@
+    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
+    let inputElemValue = inputElem.getAttribute('value');
+    hotkeys.trigger(`${mod}+shift+up`, { targetElement: inputElem });
+    let movedUpInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
+    expect(movedUpInputElem.getAttribute('value')).toEqual(inputElemValue);
+  });
+
+  it(`should move row down using 'mod+shift+down' shortcut`, () => {
+    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
+    let inputElemValue = inputElem.getAttribute('value');
+    hotkeys.trigger(`${mod}+shift+down`, { targetElement: inputElem });
+    let movedDownInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
+    expect(movedDownInputElem.getAttribute('value')).toEqual(inputElemValue);
+  });
+
+  it(`should delete the current row in table  using 'mod+backspace'' shortcut`, () => {
+    let elem = page.getChildOfElementByCss(page.getElementById('keywords.0.keyword'), 'textarea');
+    let elemBeforeDeletionThatWillBeShifted = page.getChildOfElementByCss(page.getElementById('keywords.1.keyword'), 'textarea')
+      .getAttribute('value');
+    let elemValue = elem.getAttribute('value');
+    hotkeys.trigger(`${mod}+backspace`, { targetElement: elem });
+    let elemAfterDeletionAndThatWasShifted = page.getChildOfElementByCss(page.getElementById('keywords.0.keyword'), 'textarea');
+    expect(elemAfterDeletionAndThatWasShifted.getAttribute('value')).not.toEqual(elemValue);
+    expect(elemAfterDeletionAndThatWasShifted.getAttribute('value')).toEqual(elemBeforeDeletionThatWillBeShifted);
+  });
+
+  it(`should copy new row under keywords.0 table using 'alt+c' shortcut. 
+    The copied cell under the focused one must be empty and the remaining cells must be exactly copied.`, () => {
+    let firstColInputElemValue = page.getChildOfElementByCss(page.getElementById('keywords.0.classification_scheme'), 'textarea')
+      .getAttribute('value');
+    let secondColInputElemValue = page.getChildOfElementByCss(page.getElementById('keywords.0.keyword'), 'textarea')
+      .getAttribute('value');
+    hotkeys.trigger('alt+c', { targetElement: firstColInputElemValue });
+    let firstColAfterCopyInputElemValue = page
+      .getChildOfElementByCss(page.getElementById('keywords.1.classification_scheme'), 'textarea')
+      .getAttribute('value');
+    let secondColAfterCopyInputElemValue = page.getChildOfElementByCss(page.getElementById('keywords.1.keyword'), 'textarea')
+      .getAttribute('value');
+    expect(firstColAfterCopyInputElemValue).toEqual('');
+    expect(secondColAfterCopyInputElemValue).toEqual(secondColInputElemValue);
+  });
+  it(`should copy new row under references table using 'mod+shift+c' shortcut.
```

Comment:
this fails because, `command+shift+c` is a valid Chrome shortcut on `macOS`. So we should change it.